assets/azure.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: managed-premium-retain
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: kubernetes.io/azure-disk
+reclaimPolicy: Retain
+parameters:
+  kind: Managed
+  storageaccounttype: Premium_LRS
```

Comment:
what if the VM is standard or unmanaged? Ideally the default "standard" class won't have any parameters set, so that it works with every cluster, like https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/storage-class/azure/default.yaml. Can we safely assume that openshift will only be installed on clusters where VMs are premium+managed?

Reply:
@wongma7 Azure's managed disk offering works with all of our VMs and is our recommended default. Standard storageaccounttype is meant for testing/entry level prod workloads where Premium is what we recommend for our customers for production use. I also checked and managed-premium is the closest equivalent to the aws default storage class.